expressEngine/server.js

```diff
@@ -34,7 +35,7 @@ db.open().then(async () => {
   try {
     console.log('Db connected successfully');
 
-    await app.listen(process.env.PORT);
+    await app.listen(process.env.PORT || 5000);
     console.log(`App running on port ${process.env.PORT}`);
```

Comment:
if 'PORT' is not available in .env then 'process.env.PORT' will be undefined. then 'app.listen' will use the default value(5000) and the code will run fine. but the console will not show default port number here. it will say 'undefined ' instead.

Reply:
it doesn't affect the flow anyhow

Reply:
and that might be even more useful since it will show that the port is not using env but the default value

Reply:
yup! but it does not seem to be a good approach.

Reply:
what is the correct approach for console logs?